auth-web/src/store/modules/business.ts

```diff
@@ -218,20 +218,45 @@ export default class BusinessModule extends VuexModule {
 
   // Following searchBusiness will search data from legal-api.
   @Action({ rawError: true })
-  public async searchBusiness (businessIdentifier: string): Promise<any> {
+  public async searchBusiness (businessIdentifier: string): Promise<LearBusiness> {
     const response = await BusinessService.searchBusiness(businessIdentifier)
```

Comment:
`searchBusiness` will throw an error if something went wrong. You should either add a `.catch()` call to it or wrap this in a try-catch block.

You could also do this but it doesn't really handle the error:
```
const response = await BusinessService.searchBusiness(businessIdentifier).catch(() => {})
```

Reply:
Same for the other business service calls (lines 238, 247 and 253).

At the moment, all these calls will crash the JS thread if there's a network error. The exception must be caught.

Reply:
You could also catch these errors in BusinessService and return null. Then THIS code might be OK.

Reply:
Will do